### PR TITLE
Sentiment analysis and generate songsnap summaries

### PR DESCRIPTION
**Pull request by tinghui2012**

### CODE REVIEW

File: sentiment_analysis.ipynb [hunk 1]
```diff
@@ -0,0 +1,3780 @@
...
+   },
+   "outputs": [
+    {
+     "metadata": {
+      "tags": null
+     },
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "92 : The size of tensor a (523) must match the size of tensor b (512) at non-singleton dimension 1\n",
```
Comment (sc4789) [inline sentiment_analysis.ipynb:1255]: Why do you think the notebook is getting these warnings?

File: sentiment_analysis.ipynb [hunk 2]
```diff
@@ -0,0 +1,3780 @@
...
+      "125 : The size of tensor a (515) must match the size of tensor b (512) at non-singleton dimension 1\n",
+      "459 : The size of tensor a (611) must match the size of tensor b (512) at non-singleton dimension 1\n",
+      "577 : The size of tensor a (632) must match the size of tensor b (512) at non-singleton dimension 1\n"
+     ]
+    }
+   ],
+   "source": [
+    "sentiment_pred = [\"\"]*len(output)\n",
+    "for i in range(len(output)):\n",
+    "  sentiment_pred[i] = classify_with_error(reordered_df['truncated_lyrics'].iloc[i],i)"
```
Comment (sc4789) [inline sentiment_analysis.ipynb:1265]: What is the purpose of this function?

**Comment by preethiprakash1** [comment]
Does Distilbert only require song lyrics in order to produce a sentiment?

File: sentiment_analysis.ipynb (hunk 2 above)
Comment (K-Wangaroo) [reply sentiment_analysis.ipynb:1265]: seems to be try-catching the library classify function? did you figure out which ones were causing errors?

**Comment by K-Wangaroo** [comment]
> Does Distilbert only require song lyrics in order to produce a sentiment?

i think this is the case, maybe we could add the song title in as well as some argument (because it currently isn't included in the lyrics), or we could append it to the lyrics. 

not sure if there's anything else we could add

**Pull request merged**